src/store/state/state-model.ts

```diff
@@ -167,6 +167,7 @@ export const stateModel: StateModelIF = {
     feeAcknowledgement: false,
     naics: { ...EmptyNaics },
     businessType: null,
-    businessNumber: null
+    businessNumber: null,
+    businessTypeConfirm: false
```

Comment:
I changed null to false for the initial value to make sure the unfinished step/scroll down mechanics is correct.

Reply:
It would be more correct to fix those mechanics so they return valid for businessTypeConfirm for a SP. There's already something like that for businessNumberValid that I think you can reuse.

Reply:
Fixed